docs/user/app_feature_intended.md

```diff
@@ -104,3 +105,35 @@ This is exactly the same concept as described in [Backup Repository Matching Rul
 ## Data
 
 The data provided while rendering the configuration of a device is described in the [SoT Aggregation](./app_feature_sotagg.md) overview.
+
+## Render Intended Configuration as a final push artifact
+
+The Intended Configuration Job doesn't produce a final configuration artifact ready to use to update a network device. You should understand it as the "intended" running configuration, because the main purpose is to be compared against the current running configuration (retrieved in the "backup" job).
```

Comment:
```suggestion
The Intended Configuration Job doesn't produce a final configuration artifact ready to use to update a network device. You should understand it as the "intended" running configuration, because the intended configuration attempts to generate what is in the final running configuration and not the steps to what it takes to get to that running configuration.
```